src/GitHub.Exports/Services/VSServices.cs

```diff
@@ -15,11 +17,13 @@ namespace GitHub.Services
     public class VSServices : IVSServices
     {
         readonly IGitHubServiceProvider serviceProvider;
+        readonly DTE dte = null;
 
         [ImportingConstructor]
-        public VSServices(IGitHubServiceProvider serviceProvider)
+        public VSServices(IGitHubServiceProvider serviceProvider, SVsServiceProvider vsServiceProvider)
         {
             this.serviceProvider = serviceProvider;
+            dte = (DTE)vsServiceProvider.GetService(typeof(DTE));
```

Comment:
Actually don't do this here at all. We already store the service provider in the field, retrieve the DTE service in the `TryOpenRepository` call directly so we avoid any potential problems that can cause the constructor to blow up (like, say, running this code in an integration outside of VS)